### PR TITLE
Update `pytorch_simple.py` to suggest `lr` from `suggest_loguniform`.

### PR DESCRIPTION
**Pull request by Crissman**
Since Learning Rate varies exponentially, should use `suggest_loguniform`.

### CODE REVIEW

**Comment by hvy** [comment]
Allow me to modify the title of this PR according to the contribution guidelines https://github.com/optuna/optuna/blob/master/CONTRIBUTING.md#creating-a-pull-request.
